transform/transformers/PCKTransformer.py

```diff
@@ -17,7 +19,11 @@ class PCKTransformer(object):
         },
         "139": {
             "0001": "Q01B",
+        },
+        "mci": {
+            "refresh": "RSI10B", # TODO remove this once EQ ready for switch
```

Comment:
May be worth raising a card in the backlog to cover this for when it comes it to keep visibility.